### ISSUE TITLE
Can't change source in Github Pages Settings to /docs (Step 5)

### ISSUE THREAD

**Issue by akinhwan** [KieferSivitz/vue-gh-pages#19]
Is Step 5 necessary for this deploy to work?

I was reading this article https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/

and it says 

> If your site is a User or Organization Page that has a repository named <username>.github.io or <orgname>.github.io, you cannot publish your site's source files from different locations. User and Organization Pages that have this type of repository name are only published from the master branch.

Does this mean my vue app has to be a considered a Project Page in order for` vue-gh-pages` to work?

**Comment by Capocaccia** [comment]
No your app does not have to be considered a project page. You need a master branch and a gh-pages branch.  Then on your gh-pages branch you need to go into the settings and change the Github Pages source to master branch /docs folder.

**Comment by akinhwan** [comment]
What if the dropdown is grayed out, and it only serves from master?
![selection_040](https://user-images.githubusercontent.com/10124722/38309406-05502dbe-37e8-11e8-9481-001239317b4c.png)

**Comment by Capocaccia** [comment]
Do you have a master branch and a gh-pages branch?

**Comment by skayablars** [comment]
For me, it only worked using the configuration of /docs in GitHub page in the branch master. I couldn't make it work having a /docs dir in the gh pages because GitHub only allows this in the master. If you want your GitHub page deployed from the gh pages branch you need to put the content of the /docs dir created by the package in the root of the branch.

**Comment by KieferSivitz** [comment]
I'm going to look into this over the weekend. Specifically I would like to have an option to deploy to docs or gh-pages branch as lately it seems that has caused some problems for people.

I'll be leaving this issue open and will update over the weekend after I get some time to see how doable that change will be.

**Comment by KieferSivitz** [comment]
I just wanted to update, I was working on this in conjunction with the fix to output directory but was running into some problems with the gh-pages module when switching between branches.
Now that the output directory changes have been applied I'll be able to look into what's causing those issues

**Comment by akinhwan** [comment]
Ok so now akinhwan.github.io is live, although I still have to figure out some styling issues and unresolve image paths. What i did was follow the steps in this Brad Traversy video. Basically I scrapped my repo and started from scratch. I removed git tracking by deleting the hidden .git folder in my root directory, and git cloned and pushed only from my /dist folder. Simple solution, I didn't need vue-gh-pages, or to create a gh-pages branch or any thing else overly complicated! https://www.youtube.com/watch?v=QKAHGpVEKoc
